server/common/HatoholArmPluginInterface.h

```diff
@@ -77,6 +77,7 @@ enum HapiCommandCode {
 	HAPI_CMD_GET_LAST_EVENT_ID,
 	HAPI_CMD_GET_TIME_OF_LAST_EVENT,
 	HAPI_CMD_SEND_UPDATED_TRIGGERS,
+	HAPI_CMD_SEND_RELOAD_ALL_TRIGGERS,
```

Comment:
Yes, I understand. . .
Send all trigger from the client, it is not a reload(@client).